src/routes/AccountDetailsContainer.js

```diff
@@ -87,13 +94,49 @@ class AccountDetailsContainer extends React.Component {
     return groups.filter(g => g.id === user.patronGroup)[0] || {};
   }
 
+  getItemDetails = () => {
+    const account = this.getAccount();
+    const { resources } = this.props;
+    const loanRecord = _.get(resources, ['loans', 'records'], []);
```

Comment:
```suggestion
    const loanRecords = _.get(resources, ['loans', 'records'], []);
```
It seems there are information about several records. Or do I misunderstand the idea?